### ISSUE TITLE
PCL build issue in tesseract_motion_planning

### ISSUE THREAD

**Issue by mpowelson** [ros-industrial-consortium/tesseract#128]
The issue has been fixed in 1.9.1 https://github.com/PointCloudLibrary/pcl/pull/2626. This is only an issue if you are stuck pre-1.8.1 for some reason (ie using the version that comes with Kinetic)

When POLICY CMP0074 is true and PCL_TO_FIND_COMPONENTS is false then the cmake_policy(POP) does not get called.

The solution is to do what they did in the PR. Go into PCLConfig.cmake and change
```
if(NOT PCL_TO_FIND_COMPONENTS)
  return()
endif()
```
to
```
if(NOT PCL_TO_FIND_COMPONENTS)
  if(POLICY CMP0074)
    cmake_policy(POP)
  endif()
  return()
endif()
```

**Comment by mpowelson** [comment]
@Levi-Armstrong This seems like this could effect a lot of people. I need to look into what the policy means and when you would have that enables

**Comment by mpowelson** [comment]
It looks like this bug was introduced in 1.9.0 This should only occur if you installed from source since it was fixed in 1.9.1
